### PR TITLE
[microTVM] Fix RVM onnx dependency and Zephyr document update

### PR DESCRIPTION
**Pull request by mehrdadh**
This PR make these changes:
- Adds onnx dependency to Zephyr RVM installation
- Adds README file about Zephyr reference virtual machine to keep track of current tools and versions.

### CODE REVIEW

**Comment by mehrdadh** [comment]
@areusch please take a look. Thanks!

File: apps/microtvm/pyproject.toml [hunk 1]
```diff
@@ -93,7 +93,7 @@ cffi = {version = "^1.14", optional = true}
 mxnet = {version = "^1.6.0", optional = true}
 
 # ONNX frontend
-onnx = {version = "1.6.0", optional = true}
+onnx = {version = "1.6.0", optional = false}
```
Comment (areusch) [inline apps/microtvm/pyproject.toml:96]: I wonder if we can instead change `apps/microtvm/reference-vm/zephyr/setup.sh` to do `pip install -E importer-onnx`? I'd like to eventually unify this file with the one produced by https://discuss.tvm.apache.org/t/rfc-python-dependencies-in-tvm-ci-containers/9011

Comment (mehrdadh) [reply apps/microtvm/pyproject.toml]: @areusch Thanks for the review. I moved it to "setup.sh".

**Comment by mehrdadh** [comment]
@areusch PTAL. Thanks!

**Review by areusch — APPROVED** [review]
thanks @mehrdadh !

**Pull request merged**